lms/djangoapps/commerce/management/commands/create_orders_for_old_enterprise_course_enrollment.py

```diff
@@ -0,0 +1,279 @@
+"""
+Management command to
+./manage.py lms create_orders_for_old_enterprise_course_enrollment
+./manage.py lms create_orders_for_old_enterprise_course_enrollment --start-index=0 --end-index=100
+./manage.py lms create_orders_for_old_enterprise_course_enrollment --start-index=0 --end-index=100 --batch-size=20
+"""
+
+import traceback
+from textwrap import dedent
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys.edx.keys import CourseKey
+from requests import Timeout
+from slumber.exceptions import HttpServerError, SlumberBaseException
+
+from student.models import CourseEnrollment
+from openedx.core.djangoapps.commerce.utils import ecommerce_api_client
+from enterprise.models import EnterpriseCourseEnrollment
+
+from util.query import use_read_replica_if_available
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Command to back-populate orders(in e-commerce) for the enterprise_course_enrollments.
+    """
+    help = dedent(__doc__).strip()
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        service_user = User.objects.get(username=settings.ECOMMERCE_SERVICE_WORKER_USERNAME)
+        self.client = ecommerce_api_client(service_user)
+
+    def _get_enrollments_queryset(self, start_index, end_index):
+        """
+
+        Args:
+            start_index: start index or None
+            end_index:  end index or None
+
+        Returns:
+            EnterpriseCourseEnrollments Queryset
+
+        """
+        self.stdout.write(u'Getting enrollments from {start} to {end} index (as per command params)'
+                          .format(start=start_index or 'start', end=end_index or 'end'))
+        enrollments_qs = EnterpriseCourseEnrollment.objects.filter(
+            source__isnull=True
+        ).order_by('id')[start_index:end_index]
+        return use_read_replica_if_available(enrollments_qs)
+
+    def _create_manual_enrollment_orders(self, enrollments):
+        """
+        Calls ecommerce to create orders for the manual enrollments passed in.
+
+        Returns (success_count, fail_count)
+        """
+        try:
+            order_response = self.client.manual_course_enrollment_order.post(
+                {
+                    "enrollments": enrollments
+                }
+            )
+        except (SlumberBaseException, ConnectionError, Timeout, HttpServerError) as exc:
+            self.stderr.write(
+                "\t\t\tFailed to create order for manual enrollments for the following enrollments: {}. Reason: {}"
+                .format(enrollments, exc)
+            )
+            return 0, 0, len(enrollments), []
+
+        order_creations = order_response["orders"]
```

Comment:
No need for the local variable. Just reference the orders directly in the for loop below.